Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -93,7 +93,7 @@ void IOnClickListener.OnClick(AView v)
 		public override bool OnInterceptTouchEvent(MotionEvent ev)
 		{
 			if (Element.InputTransparent && Element.IsEnabled)
-				return false;
+				return true;
```

Comment:
Ok. So, this is what I think you're saying. The `Button` is a child of the `ViewGroup`. It receives a touch event. That event hits this method first to see if it should flow through to the `Button`. Previously, if the `Element` was `InputTransparent`, it _would_ flow through to the `Button`. But the `Button` did not handle `onTouch`, so it wouldn't actually respect the `InputTransparent` value. Now, this `ViewGroup` will intercept the event and use _its_ `onTouch` handler, which _does_ take the `InputTransparent` value into account. Cool.